extensions/wikia/ArticleVideo/scripts/recommended-video.jwplayer.js

```diff
@@ -118,6 +121,12 @@ require([
 	}
 
 	function init() {
+		var isInAutoplayGroup = window.Wikia.AbTest.inGroup('RECOMMENDED_VIDEO_AB', 'AUTOPLAY'),
+			isInClickToPlayGroup = window.Wikia.AbTest.inGroup('RECOMMENDED_VIDEO_AB', 'CLICK_TO_PLAY');
+
+		isEnabled = isInAutoplayGroup || isInClickToPlayGroup;
+		isAutoplay = isInAutoplayGroup;
```

Comment:
how about `isAutoplay = inGroup('RECOMMENDED_VIDEO_AB', 'AUTOPLAY')`
We don't need two variables. Similar with `isEnabled`. It's only used once.